### DISCUSSION TITLE
[RFC Call] Agenda - December 28, 2021

### DISCUSSION THREAD

**Discussion by matthewp** [withastro/roadmap#60] [Proposal]
# Agenda

* @matthewp - attempt consensus: https://github.com/withastro/rfcs/pull/46

--- 

_If there is something that you would like to see discussed, please comment below. We will prioritize pending RFC, but if we have time to spare we will open the call up to get attention on and discuss [other threads and feature requests.](https://github.com/withastro/rfcs/discussions)_
If there is something else you would like to discuss, please mention it here.

**Comment by matthewp** [comment]
We had extra time and wanted to talk about import assertions...

```js
import package from '/package.json' assert { type: 'json' };
import package from '/package.json';
```

---

```astro
---
import scriptUrl from '/script.js' assert { type: 'url' };
import scriptUrl from '/script.js?url';
import script from '/script.js';
--

<script src="local:./script.js"></script>
<script src={await import('./script.js?url')}></script>

<img srcset="2x local:./penguin.png" />
<img srcset={`2x ${await import('./penguin.png')}`} />
```

---

**Comment by hans-d** [comment]
part of discussion, but more process related.

Proposal to explicitly include in "new" rfcs: will it introduce ssr blockers, does it remove ssr blockers or will it be ssr neutral.
